### PR TITLE
Added tests on properties and link tracking pattern

### PR DESCRIPTION
**Pull request by umbobabo**

### CODE REVIEW

File: config_revamp.es6 [hunk 1]
```diff
@@ -1,13 +1,14 @@
 /* eslint-disable id-match */
 import slugger from 'slugger';
 import User from '@economist/user';
+//import OmnitureUtils from '@economist/react-i13n-omniture/OmnitureUtils';
 import OmnitureUtils from './OmnitureUtils';
 function slug(string) {
   return slugger(String(string || ''), { replacement: '_' });
 }
-
+const isProductionEnv = (process.env.ENV === 'prod' && process.env.NODE_ENV === 'production');
```
Comment (keithamus) [inline config_revamp.es6]: Just NODE_ENV should be sufficient.

File: config_revamp.es6 [hunk 2]
```diff
@@ -1,13 +1,14 @@
 /* eslint-disable id-match */
 import slugger from 'slugger';
 import User from '@economist/user';
+//import OmnitureUtils from '@economist/react-i13n-omniture/OmnitureUtils';
```
Comment (keithamus) [inline config_revamp.es6]: Please don't add commented out code :smile:

File: config_revamp.es6 [hunk 3]
```diff
@@ -1,13 +1,14 @@
 /* eslint-disable id-match */
 import slugger from 'slugger';
 import User from '@economist/user';
```
Comment (keithamus) [inline config_revamp.es6]: If we're keeping this as a clean omniture, then we shouldn't have this here - it should be upstream.

File: config_revamp.es6 [hunk 4]
```diff
@@ -48,11 +49,26 @@ const OmnitureConfig = {
...
+      }
+      if (nodeProps.element){
+        linkInfo.push(slug(nodeProps.element))
+      }
+      const prop45 = linkInfo.join('>');
+      return {
+        linkType: nodeProps.product,
+        linkName: nodeProps.element,
+        prop45
+      }
```
Comment (keithamus) [inline config_revamp.es6]: This whole block looks like its specific to our use case of omniture, so should not be in this plugin.

File: config_revamp.es6 [hunk 5]
```diff
@@ -101,25 +117,35 @@ const OmnitureConfig = {
...
         prop10: OmnitureUtils.fullDate(),
         prop11: OmnitureUtils.userLoggedIn(),
-        prop13: OmnitureUtils.userSubscription(),
+        prop13: userSubscription,
         prop31: OmnitureUtils.articlePublishDate(nodeProps.publishDate),
         prop32: location.href,
         prop34: OmnitureUtils.deviceDetection(),
-        prop40: '',
+        prop40: User.getUserId(),
+        prop46: User.isMultiUserLicense() ? 'MUL-IP' : '',
```
Comment (keithamus) [inline config_revamp.es6]: Aren't these overridden by the instantiation of the plugin? Again though, feels like this should be upstream.

File: config_revamp.es6 (hunk 1 above)
Comment (umbobabo) [inline config_revamp.es6]: Unfortunately not.
On stage process.env.NODE_ENV==='production' and they don't want to use the same Omniture account for stage.

File: config_revamp.es6 (hunk 3 above)
Comment (umbobabo) [inline config_revamp.es6]: The fact is that this part is duplicate on the fe-blogs repo because it context related more than plugin related.
But  after talked with @sublimino we agreed that was simpler write the tests here due to how the tests run in relationship to the client side nature of this part.

File: config_revamp.es6 (hunk 4 above)
Comment (umbobabo) [inline config_revamp.es6]: As I said before this file is here because it currently easier.
Any different solution will be appreciated.
Anyway this file is not related to the pure BL of the plugin.

File: config_revamp.es6 (hunk 1 above)
Comment (keithamus) [inline config_revamp.es6]: This should probably be overridable via environment variable then, right?

Comment (fabiosantoscode) [inline config_revamp.es6]: Can't see the use case. Environment variables are being passed through fe-blogs, and we'd like to keep them at a minimum. Saying that ENV=stage or ENV=ci should be enough to say what account ID to use.

It's not like this is going to change anytime soon anyway.

File: config_revamp.es6 (hunk 5 above)
Comment (fabiosantoscode) [inline config_revamp.es6]: Absolutely. 

This config file is only used in some tests which shouldn't be here anyway.

File: test/index.es6 [hunk 6]
```diff
@@ -139,3 +127,155 @@ describe('OmniturePlugin is a i13n plugin for Omniture', () => {
...
+        const pageviewOutput = RevampConfig.eventHandlers.pageview({
+          articleSource: 'web',
+        });
+        pageviewOutput.should.have.property('prop13', 'digital-subscriber|none');
+        pageviewOutput.should.have.property('eVar13', 'digital-subscriber|none');
+      });
+    });
+    describe('it return different value for different pages', () => {
+      it('it return a value for blog page list', () => {
+        const pageviewOutput = RevampConfig.eventHandlers.pageview({
```
Comment (fabiosantoscode) [inline test/index.es6]: As we were talking about yesterday, these tests need to be in fe-blogs, and they should be browser-driving tests that use real data from the website.

Nuke everything that uses RevampConfig and put it somewhere safe so we can turn them into browser tests using real data.

**Comment by fabiosantoscode** [comment]
I'm closing this pull request now that these tests are on fe-blogs, as a mixture of integration and unit tests.
